robot_skill:
  ros__parameters:
    # 留空表示使用默认的 package skill_sets 目录，若需自定义在此填写绝对路径
    skill_sets_dir: ''
